components/JobCard.svelte

```diff
@@ -0,0 +1,119 @@
+<script>
```

Comment:
This one is too specific to fit in a reusable library. Any of the components that depend on data that is specific to an app fit this description (e.g. gig, job, language, etc.)